WordPress/gutenberg
# Feat: Add border color option to button block

## What?

- This PR would add an option to add `border` color to `buttons` block. So user can change border color anytime for any buttons they use on the site.

## Why?

Issue refrence - [#43072](https://github.com/WordPress/gutenberg/issues/43072)

## How?

- This PR changes the `block.json` file for button block and adds the support for `border color` to it.

## Testing Instructions




1. Open any post/page.
2. Add `Buttons` block.
3. You can check the block controls option in sidebar having option to choose color for border.
4. Explicitly, you can also change the border width as this comes along with color option.

### Testing Instructions for Keyboard

You can use Tabs to change color on newly option created.

## Screenshots or screencast 

<img width="1440" alt="Screenshot 2023-01-24 at 5 27 00 PM" src="https://user-images.githubusercontent.com/58802366/214285668-e54c5074-3932-4885-9817-3a9711956cf3.png">
